frontend/app/src/components/history/filtering/SuggestedItem.vue

```diff
@@ -25,10 +25,12 @@ const displayValue = computed(() => {
     return value;
   }
 
+  const symbol = value.isCustomAsset ? value.name : value.symbol;
```

Comment:
Maybe we can move symbol after the if or inline it in the last return. Do reason to evaluate this and assign to a const if it is an evm chain right?

Reply:
Updated